### PR TITLE
chore: remove `securityCouncilTokenOwner` from deploy config

### PR DESCRIPTION
**Pull request by kangsorang**
# Description

Since we do not need `securityCouncilTokenOwner` anymore, remove it and use `proxyAdminOwner` for `SystemConfig` owner in e2e tests.

### CODE REVIEW

**Comment by kangsorang** [comment]
I've split the transferOwnership content into a separate task.

**Review by kangsorang — COMMENTED** [review]
LGTM

File: e2e/e2eutils/setup.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ func MakeDeployParams(t require.TestingT, tp *TestParams) *DeployParams {
 		DeploymentWaitConfirmations: 1,
 		ValidatorRewardScalar:       5000,
 
-		SecurityCouncilTokenOwner:    addresses.SysCfgOwner,
+		ProxyAdminOwner:              addresses.SysCfgOwner,
```
Comment (0xHansLee) [inline e2e/e2eutils/setup.go:118]: IMO, using both `ProxyAdminOwner` and `SysCfgOwner` could be confusing. 
Why don't we change `SysCfgOwner` to `ProxyAdminOwner` as well?

Comment (seolaoh) [reply e2e/e2eutils/setup.go]: I added one more commit applying it. Please check it.

**Review by seolaoh — APPROVED** [review]

**Review by 0xHansLee — APPROVED** [review]
lgtm

**Pull request merged**